composer.json

```diff
@@ -26,12 +26,12 @@
         "sort-packages": true
     },
     "require": {
-        "php": "^7.1"
```

Comment:
would simply go with `>=7.1`.

objections?

Reply:
Well, it's true that you'd never have to change it again! I think it's good to keep this aligned with the versions that you're actually testing against.